tools/cli/commands.go

```diff
@@ -271,41 +273,66 @@ func UpdateDomain(c *cli.Context) {
 		}
 		if c.IsSet(FlagClusters) {
 			clusterStr := c.String(FlagClusters)
-			clusters = append(clusters, &s.ClusterReplicationConfiguration{
+			clusters = append(clusters, &shared.ClusterReplicationConfiguration{
 				ClusterName: common.StringPtr(clusterStr),
 			})
 			for _, clusterStr := range c.Args() {
-				clusters = append(clusters, &s.ClusterReplicationConfiguration{
+				clusters = append(clusters, &shared.ClusterReplicationConfiguration{
 					ClusterName: common.StringPtr(clusterStr),
 				})
 			}
 		}
 
-		updateInfo := &s.UpdateDomainInfo{
+		var binBinaries *shared.BadBinaries
+		if c.IsSet(FlagAddBadBinary) {
+			if !c.IsSet(FlagReason) {
+				ErrorAndExit("Must provide a reason.", nil)
+			}
+			binChecksum := c.String(FlagAddBadBinary)
+			reason := c.String(FlagReason)
+			operator := getCurrentUserFromEnv()
```

Comment:
We may still need optional flag for this, otherwise open source user have no clue on how to set operator

Reply:
That was my initial proposal.  But we decided not to expose it. If they really want to set operator, then can set it through env var.